addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -13,22 +13,23 @@ function ActivityFactory({ Entity }) {
 
     class Activity extends Entity {
 
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
         /**
-         * @override
+         * Delete the record from database and the entity.
```

Comment:
Just a bit more clear:
```suggestion
         * Delete the record from database and delete the entity.
```